modules/apps/27-interchain-accounts/genesis/types/genesis.go

```diff
@@ -77,7 +77,7 @@ func (gs ControllerGenesisState) Validate() error {
 		}
 	}
 
-	return gs.Params.Validate()
+	return nil
```

Comment:
Why was the validation of params removed? For host params the validation is still there.

Reply:
Validation is not needed for parameters that are just booleans. Because all booleans are valid. Validation was also removed from transfer. The validation was needed when we used `x/params` in case the external module did something funny. In host module, the params weren't just bools